src/utils/junitUtils.ts

```diff
@@ -0,0 +1,40 @@
+import { SendJsonOutput } from '@/cli/send/jsonOutput';
+
+export function toJunitXml(output: SendJsonOutput): string {
+  let resultXml = `
+  <?xml version="1.0" encoding="UTF-8"?>
+  <testsuites 
+    disabled="0"
+    errors="${output.requests.flatMap(r => r.testResults).filter(r => r?.error !== undefined).length}"
+    failures="${output.summary.failedTests}"
+    tests="${output.summary.totalTests}"
+    time="0">
+`;
+  for (const req of output.requests) {
+    resultXml += `
+    <testsuite
+      name="${req.fileName}"
+      tests="${req.summary.totalTests}"
+      disabled="0"
+      errors="${req.testResults?.filter(r => r.error !== undefined).length}"
```

Comment:
I check for r.result, https://github.com/AnWeber/httpyac/blob/main/src/cli/send/jsonOutput.ts#L63
```suggestion
      errors="${req.testResults?.filter(r => °r.result).length}"
```